test_data/config_demo.json

```diff
@@ -182,6 +182,42 @@
         }
       }
     },
+    {
+      "type": "FeatureTrack",
```

Comment:
Should this be a VariantTrack?

Reply:
yep perils of manually edited configs. the featuretrack works for it though since we don't really have too much custom varianttrack functionality for this one

Reply:
Yeah, the only differences I can think of would be the available displays and the feature widget, although I don't think this VCF has samples so the variant feature widget might not even be look different from the base feature widget.